up42/jobcollection.py

```diff
@@ -33,10 +33,69 @@ def __repr__(self):
             f"auth={self.auth})"
         )
 
-    # TODO: Make class subscriptable.
+    def __getitem__(self, index: int) -> Job:
+        return self.jobs[index]
+
+    def __iter__(self):
+        for job in self.jobs:
+            yield job
+
+    def apply(
```

Comment:
Probably private method best, also to avoid in docs. Or do you see any usage by the user outside of the provided functions?

Reply:
I think makes sense to make it public, so users could built custom loops if they want, like getting all job params for instance. We could implement more but if it's public users can also do that.